model/res/src/main/scala/aqua/res/MakeRes.scala

```diff
@@ -62,26 +62,31 @@ object MakeRes {
     case NextModel(item) => NextRes(item).leaf
     case PushToStreamModel(operand @ VarModel(_, StreamType(st), _), exportTo) =>
       val tmpName = s"push-to-stream-$i"
+      val property = operand.properties
       // wrap (
       //  RestrictionRes(tmpName, isStream = false),
       SeqRes.wrap(
-        canon(
+        CanonRes(
+          operand.copy(properties = Chain.empty),
           orInit(currentPeerId),
-          operand,
-          CallModel.Export(tmpName, ArrayType(st))
-        ),
-        ApRes(VarModel(tmpName, ArrayType(st), Chain.empty), exportTo).leaf
+          CallModel.Export(tmpName, CanonStreamType(st))
+        ).leaf,
+        ApRes(VarModel(tmpName, CanonStreamType(st), property), exportTo).leaf
```

Comment:
What if `property` (should be `properties`, I suppose) is empty? Isn't this block useless?